0000-template.md

```diff
@@ -1,3 +1,14 @@
+<!---
+Copyright <fill in year> The Rust Project Developers. See the COPYRIGHT
```

Comment:
Please don't use "Copyright YYYY The Rust Project Developers. That isn't a legal entity. In the absence of copyright assignments (which don't seem like a good idea, and which the Rust project doesn't use in general), the copyrights would be held by the individual developers.

If you don't want to have people list individual copyright notices, you could just leave this part out and just keep the second paragraph. But please don't create a legally ambiguous situation by listing "The Rust Project Developers" as the copyright holder.

(You might, in an appropriate top-level file, note that copyrights are retained by the individual contributors. Listing every copyright holder is not strictly necessary from a legal perspective, though.)